src/main/resources/db/changelog/test.sql

```diff
@@ -1 +1,73 @@
---liquibase formatted sql
\ No newline at end of file
+--liquibase formatted sql
+
+--changeset k4t:002_1 context:"test"
+create table if not exists users (
+                                     id       uuid not null primary key,
+                                     nickname varchar(255),
+    created_at timestamp,
+    email varchar(120),
+    password varchar(20),
+    role varchar(255),
+    status varchar(255)
+    );
+alter table users
+alter column password type varchar;
+
+--changeset k4t:002_2 context:"test"
+create table if not exists categories (
+                                          id   uuid not null primary key,
+                                          name varchar(255)
+    );
+
+--changeset k4t:002_3 context:"test"
+create table if not exists quizzes (
+                                       id          uuid not null primary key,
+                                       name        varchar(255),
+    category_id uuid,
+    user_id     uuid,
+    created_at timestamp
+    );
+
+--changeset k4t:002_4 context:"test"
+create table if not exists questions (
+                                         id      uuid not null primary key,
+                                         content varchar(255),
+    quiz_id uuid
+    );
+
+--changeset k4t:002_5 context:"test"
+create table if not exists answers (
+                                       id          uuid not null primary key,
+                                       content     varchar(255),
+    is_correct  boolean,
+    question_id uuid
+    );
+
+--changeset k4t:002_6 context:"test"
+create table if not exists game_user_answers (
+                                                 game_id uuid,
+                                                 user_answers uuid
+);
+
+--changeset k4t:002_7 context:"test"
+create table if not exists games (
+                                     id uuid not null,
+                                     quiz_id uuid,
+                                     created_at timestamp,
+                                     primary key(id)
+    );
+
+--changeset k4t:002_8 context:"test"
+create table if not exists results (
+                                       id uuid not null,
+                                       score integer,
+                                       game_id uuid,
+                                       quiz_id uuid,
+                                       primary key(id)
+    );
+
+--changeset k4t:002_13 context:"test"
```

Comment:
Why do we have the changeset number 002_8 above and 002_13 number here ?

Reply:
during trial and error, I deleted a few entries